<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7BBMSHU" failures="1" tests="3" name="Test" time="5.58" errors="0" timestamp="2025-01-30T06:25:37 IST">
  <testcase classname="MyPrograms.TestNGProgram" name="failMethod" time="3.452">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at MyPrograms.TestNGProgram.failMethod(TestNGProgram.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- failMethod -->
  <testcase classname="MyPrograms.TestNGProgram" name="passMethod" time="1.626"/>
  <testcase classname="MyPrograms.TestNGProgram" name="skipTestConditionally" time="0.008">
    <skipped/>
  </testcase> <!-- skipTestConditionally -->
</testsuite> <!-- Test -->
